lib/parliament/builder/base_response_builder.rb

```diff
@@ -1,12 +1,21 @@
 module Parliament
   module Builder
+    # API response builder, allowing the user to return the body of an HTTPResponse.
+    # @since 0.7.5
     class BaseResponseBuilder
-      def initialize(response)
+      # Creates a new BaseReponseBuilder.
+      # @param [HTTPResponse] response an HTTP response containing n-triple data.
+      # @param [Module] decorators the decorator modules to provide alias methods to the resulting objects.
```

Comment:
Infact, is this even needed? Our base response doesn't actually use the decorators?